avocado_vt/loader.py

```diff
@@ -16,6 +16,7 @@
 Avocado VT plugin
 """
 
```

Comment:
One first thing I have to comment is based on the commit message alone.  The creation of a shared function to configure argparse for both vt and vt-list plugins is a refactor, as it makes the core more optimal but *should not* change any behavior.  Then you add a new option, which changes behavior.

You should really split those two, very different things, into their own commits.